Complications/README.md

```diff
@@ -5,6 +5,11 @@ Android Complications Sample
 Complication Test Suite is a set of complication data sources that provide dummy data and it can be
 used to test how different types of complications render on a watch face.
 
+> [!IMPORTANT]  
+> As of July 10, 2024, watch faces must use the [Watch Face Format][1] ([sample][3]) in order to be installed on new watches that launch with Wear OS 5 pre-installed. Learn more about the user-facing changes in this [Help Center][2] article.
```

Comment:
Incorrect recycling of reference links (which we use at the end of the introduction)

Reply:
I think I've fixed this!